perly.y

```diff
@@ -709,7 +709,11 @@ proto
 subattrlist
 	:	empty
 	|	COLONATTR THING
-			{ $$ = $THING; }
+			{
+			  if($THING && !PL_parser->sig_seen)
+			      $THING = apply_builtin_cv_attributes(PL_compcv, $THING);
```

Comment:
better assign directly to `$$` than to named reference

Reply:
Oh, fair. Actually probably better yet is to make a local variable out of it;

```C
OP *attrlist = $THING;
...
```

Reply:
Adjusted and force-pushed. PTAL